package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@yext/chat-core",
-  "version": "0.7.4",
+  "version": "0.7.5",
```

Comment:
can you do `npm i` to update package-lock.json as well

Reply:
done